lib/instrumentation/modules/mongodb.js

```diff
@@ -19,7 +19,7 @@ const HOSTNAME_PORT_RE = /^(.+):(\d+)$/;
 
 module.exports = (mongodb, agent, { version, enabled }) => {
   if (!enabled) return mongodb;
-  if (!semver.satisfies(version, '>=3.3 <7.0')) {
+  if (!semver.satisfies(version, '>=3.3 <7')) {
```

Comment:
Note for reviewer: THis change is a necessary part of this, but was a style change I made to match our typical `... <$majorVer` style that we use in module instrumentations when I was reviewing them all.